contracts/contracts.go

```diff
@@ -11,8 +11,9 @@ import (
 	"github.com/0xsequence/go-sequence/contracts/gen/ierc1271"
 	"github.com/0xsequence/go-sequence/contracts/gen/niftyswap"
 	seqmarketplace "github.com/0xsequence/go-sequence/contracts/gen/seq_marketplace"
-	seqsale1155 "github.com/0xsequence/go-sequence/contracts/gen/seq_sale/erc1155"
-	seqsale721 "github.com/0xsequence/go-sequence/contracts/gen/seq_sale/erc721"
+	seqsale1155v0 "github.com/0xsequence/go-sequence/contracts/gen/seq_sale/erc1155v0"
+	seqsale1155v1 "github.com/0xsequence/go-sequence/contracts/gen/seq_sale/erc1155v1"
+	seqsale721v0 "github.com/0xsequence/go-sequence/contracts/gen/seq_sale/erc721v0"
```

Comment:
Hi, @ScreamingHawk Can you please review this PR?

It looks like we need some guidance on the semver versioning of the contracts.

I might be looking wrong but I don't see any versions at https://github.com/0xsequence/contracts-library. Can you help me understand how do we version the contracts right now? Is there any page (or source of truth) with the list of contracts and their addresses / versions when you deploy them on chains?

Reply:
I wonder who should be responsible for maintaining https://github.com/0xsequence/go-sequence/tree/master/contracts/artifacts and when we introduce any breaking changes, how do we upgrade repos like `stack/api` / `relayer` / `marketplace` etc.

Sorry I'm new to this, so I might be missing something obvious :D

Reply:
I consider the Builder database is the source of truth. 
Updating these artifacts could be my (the contract developer) responsibility. 
I have a notion page with information on what to update with a new contract deployment but it's missing a lot of details. I'm also going to start pinging in Slack for new versions so everyone is aware.